diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -1677,170 +1586,119 @@ export class ContentGame extends React.Component {
 
         const convList = tabNames.map((protagonist) => (
-                <Conversation style={{ minWidth: "200px" }}
-                    info={
-                        protagonist !== "GLOBAL"
-                            ? this.truncate(
-                                  engine.powers[protagonist].getController()
-                              )
-                            : ""
-                    }
-                    unreadCnt={unreadCnt(protagonist, currentTabId)} info={protagonist!=='GLOBAL' ? engine.powers[protagonist].getController():""}
-                    className={
-                        protagonist === currentTabId
-                            ? "cs-conversation--active"
-                            : null
-                    }
-                    title={
-                        protagonist !== "GLOBAL"
-                            ? engine.powers[protagonist].getController()
-                            : ""
-                    }
-                    onClick={() => {
-                        this.onChangeTabCurrentMessages(protagonist);
-                    }}
-                    key={protagonist}
+            <Conversation
+                style={{ minWidth: "200px" }}
+                info={
+                    isAdmin && protagonist !== "GLOBAL"
+                        ? engine.powers[protagonist].getController()
+                        : ""
+                }
+                className={
+                    protagonist === currentTabId
+                        ? "cs-conversation--active"
+                        : null
+                }
+                onClick={() => {
+                    this.onChangeTabCurrentMessages(protagonist);
+                }}
+                key={protagonist}
+                name={protagonist}
+                unreadCnt={this.countUnreadMessages(
+                    engine,
+                    role,
+                    protagonist
+                )}
+                unreadDot={this.hasUnreadAdvice(engine, role, protagonist)}
+            >
+                <Avatar
+                    src={POWER_ICONS[protagonist]}
                     name={protagonist}
-                    unreadCnt={this.countUnreadMessages(
-                        engine,
-                        role,
-                        protagonist
-                    )}
-                >
-                    <Avatar
-                        src={POWER_ICONS[protagonist]}
-                        name={protagonist}
-                        size="sm"
-                        status={protagonist!=='GLOBAL' ? (engine.powers[protagonist].getCommStatus()===STRINGS.READY ? "available":"dnd"):"invisible"}
-                    />
-                </Conversation>
+                    size="sm"
+                    status={
+                        isAdmin && protagonist !== "GLOBAL"
+                            ? engine.powers[protagonist].getCommStatus() ===
```

Comment:
Because it's still useful to see during debugging. I tested, and it shouldn't be showing up for players at all.

There is an off-white dot shown near the player icon, but that seems to be part of the library and difficult to hide.